src/Tribe/Asset/Script_Data.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Handles adding script data to the page in cases where localizing a
+ * specific script is not suitable.
+ *
+ * Should generally be accessed via tribe( 'tribe.asset.script-data' )
+ * rather than via direct instantiation.
+ */
+class Tribe__Asset__Script_Data {

```

Comment:
I would rename it to `Tribe__Asset__Data` since no CSS would have data attached to it.